CONTRIBUTING.md

```diff
@@ -56,8 +56,9 @@ cannot be merged.
 Do:
 * enable strings visible in the Mudlet GUI to be translateable
 * minimise use of HTML styling tags in strings to be translated
+* enable users to use language-specific Mudlet object names (triggers, aliases, labels, etc)
 
 Don't:
-* translate the Mudlet API: functions, events, or error messages
+* translate the Mudlet API: functions, events, error messages or constants (`main` console, mapper directions)
```

Comment:
"mapper directions" ? - you may have to be a bit more specific/focused as one of the things we WILL have to add is a system to customise the strings that Mudlet produces for speed-walking and those WILL need to have language specific, but over-ridable, defaults to use...!

OTOH any new function added that takes or produces exit directions WILL need to be coded with care to fit in with existing methods!